Dockerfile

```diff
@@ -15,29 +12,55 @@ RUN cd /root/bcc && \
     /usr/lib/pbuilder/pbuilder-satisfydepends && \
     PARALLEL=$(nproc) ./scripts/build-deb.sh release
 
-FROM ubuntu:bionic
+RUN cd /root && git clone https://github.com/libbpf/libbpf.git && \
+    cd /root/libbpf && git checkout 030ff87857090ae5c9d74859042d05bfb3b613a2  && cd src && mkdir build root && BUILD_STATIC_ONLY=y OBJDIR=build DESTDIR=root make install
+
+FROM ubuntu:20.04 as compiler
+ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends git build-essential libelf1 software-properties-common
```

Comment:
Please make sure that Dockerfile is still operational:

```
Step 18/32 : RUN cd /go/ebpf_exporter && CC=gcc CGO_CFLAGS="-I /usr/include/bpf" CGO_LDFLAGS="-lbpf" GOPATH="" GOPROXY="off" GOFLAGS="-mod=vendor" go install -v -ldflags="     -X github.com/prometheus/common/version.Version=$(git describe)     -X github.com/prometheus/common/version.Branch=$(git rev-parse --abbrev-ref HEAD)     -X github.com/prometheus/common/version.Revision=$(git rev-parse --short HEAD)     -X github.com/prometheus/common/version.BuildUser=docker@$(hostname)     -X github.com/prometheus/common/version.BuildDate=$(date --iso-8601=seconds)     " ./cmd/ebpf_exporter
 ---> Running in 7f291d3a0ba1
/bin/sh: 1: git: not found
/bin/sh: 1: git: not found
/bin/sh: 1: git: not found
```

Reply:
Fixed

Reply:
I'm getting a linker error now:

```
# runtime/cgo
/usr/bin/ld: cannot find -lbpf
collect2: error: ld returned 1 exit status
```

Not sure if the fact that I'm on aarch64 has anything to do with it.